Nike-Inc/koheesio
# [DOC] documentation updates June 2024

Documentation updates

## Description
- Fixed spelling of and type annotations for Parameters across docstrings
- Fixed broken links and unrecognized relative links
- Fixed indentation for continuation lines in docstrings
- Refactored README for legibility and clarity with a more defined structure and better formatting
- General style and formatting improvements
- Minor change to the layout of the pages (to better accommodate the reader)
- No code changes were made as part of this PR

## Testing
To build the docs: 
- check out the branch
- run `make docs`
- docsite will be hosted on your local machine

## Related Issue
#38 
#42 

## Types of changes


## Checklist:
